src/common/transformations/src/transformations/common_optimizations/nop_elimination.cpp

```diff
@@ -290,7 +290,8 @@ static bool eliminate_unsqueeze(const shared_ptr<Node>& node) {
 
 SIMPLE_MATCHER_PASS_DEFINITION(EliminateReshape, opset3::Reshape, eliminate_reshape_v1);
 SIMPLE_MATCHER_PASS_DEFINITION(EliminateUnsqueeze, opset3::Unsqueeze, eliminate_unsqueeze);
-SIMPLE_MATCHER_PASS_DEFINITION(EliminateBroadcast, op::v1::Broadcast, eliminate_nop);
+SIMPLE_MATCHER_PASS_DEFINITION(EliminateBroadcast_v1, op::v1::Broadcast, eliminate_nop);
```

Comment:
Alternatively we could have something like:
`SIMPLE_MATCHER_PASS_DEFINITION(EliminateBroadcast, eliminate_nop, op::v1::Broadcast, op::v3::Broadcast);`
but that would imply changes to SIMPLE_MATCHER_PASS_DEFINITION macro (like `#define SIMPLE_MATCHER_PASS_DEFINITION(NAME, FUNC, ...)`) and using `__VA_ARGS__`  when calling wrap_type function.

Reply:
Done